MAINTAINERS

```diff
@@ -1,8 +1,58 @@
-Solomon Hykes <solomon@docker.com> (@shykes)
-Olivier Gambier <olivier@docker.com> (@dmp42)
-Stephen Day <stephen.day@docker.com> (@stevvooe)
-Derek McGowan <derek@mcgstyle.net> (@dmcgowan)
-Richard Scothern <richard.scothern@gmail.com> (@richardscothern)
-Aaron Lehmann <aaron.lehmann@docker.com> (@aaronlehmann)
+# Distribution maintainers file
+#
+# This file describes who runs the docker/distribution project and how.
+# This is a living document - if you see something out of date or missing, speak up!
+#
+# It is structured to be consumable by both humans and programs.
+# To extract its contents programmatically, use any TOML-compliant parser.
+#
+# This file is compiled into the MAINTAINERS file in docker/opensource.
+#
+[Org]
+	[Org."Core maintainers"]
```

Comment:
We are standardizing the format so we can keep in one file in docker/opensource. This was originally to clean up the maintainers in the docker/docker repo itself because only maintainers of that repo should be in there but it was being used as a catch all for all docker projects. Now the consolidated file can be used to make sure LGTMs come from valid maintainers. There is an automated way of parsing all the repos files. And people who want to see a considered list of maintainers can without the docker/docker repo needing to be this outdated catch all. It was confusing that way. And now we can clean up the people who even have merge rights on specific repos. It's unified across docker projects, and lgtm.co even integrated it into their parsing, but as it says in the rules in docker/opensource, if you don't like it, make a pull request to change it :)






On Tue, Dec 8, 2015 at 11:57 AM -0800, "Stephen Day" <notifications@github.com> wrote:












In MAINTAINERS:
> -Stephen Day <stephen.day@docker.com> (@stevvooe)
> -Derek McGowan <derek@mcgstyle.net> (@dmcgowan)
> -Richard Scothern <richard.scothern@gmail.com> (@richardscothern)
> -Aaron Lehmann <aaron.lehmann@docker.com> (@aaronlehmann)
> +# Distribution maintainers file
> +#
> +# This file describes who runs the docker/distribution project and how.
> +# This is a living document - if you see something out of date or missing, speak up!
> +#
> +# It is structured to be consumable by both humans and programs.
> +# To extract its contents programmatically, use any TOML-compliant parser.
> +#
> +# This file is compiled into the MAINTAINERS file in docker/opensource.
> +#
> +[Org]
> +	[Org."Core maintainers"]




It's already working w 10+ other repos :p




Sorry to ask questions. I have no clue what this is and there is no explanation as to why we would replace a six line file with a 100+ line file to do the exact same thing. If no one asked that question yet, it should probably be answered.



—
Reply to this email directly or view it on GitHub.